.github/workflows/lighthouse-prod.yml

```diff
@@ -19,6 +19,10 @@ jobs:
       PSI_API_KEY: ${{ secrets.LHCI_PSI_API_KEY }}
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 40
```

Comment:
I went with a larger number here, rather than 20 as mentioned, b/c sometimes we have more than 20 commits in a day...